config/samples/secret-agent_v1alpha1_secretagentconfiguration.yaml

```diff
@@ -102,25 +115,78 @@ spec:
         storePassPath: am-passwords/.storepass
         keyPassPath: am-passwords/.keypass
         keytoolAliases:
-          - name: rsajwtsigningkey
-            cmd: genkeypair
-            args: ["-keyalg", "RSA", "-keysize", "2048", "-validity", "7300"]
-            destinationPath: truststore/cacerts
-          - name: ec256test
-            cmd: genkeypair
-            args : ["-keyalg", "EC","-sigalg",  "SHA256withECDSA", "-dname", "CN=ForgeRock", "-keysize", "256"]
-            destinationPath: truststore/cacerts
-          - name: hmacsigningtest
-            cmd: genseckey
-            args: ["-keyalg", "HMacSHA512", "-keysize", "512"]
-            destinationPath: truststore/cacerts
-          - name: configstorepwd
-            cmd: importpassword
-            sourcePath: am-passwords/.storepass
-          - name: my-imported-cert
-            cmd: importcert
-            sourcePath: truststore/cacerts
+        - name: rsajwtsigningkey
```

Comment:
I haven't looked yet but, I'm guessing you got all these from the forgeops-secrets? Is there anything left to be implemented for the key `am-keystore`?

Reply:
The sms stuff is still pending
`keytool -importkeystore -destalias sms.transport.key -srcalias sms.transport.key -srcstoretype jceks -srckeystore sms-transport-key/sms-transport-key.jceks -srckeypass:file sms-transport-key/keypass -srcstorepass:file sms-transport-key/storepass -destkeystore /opt/gen/secrets/generic/am-keystore/keystore.jceks -deststoretype jceks -deststorepass zzggthedwwd4pegt1tth02drmxfbdw01 -destkeypass is4spw1bfagasmsc0gpgd2gwrgrzchz0`